ontology/yaml/resources/HVAC/entity_types/MAU.yaml

```diff
@@ -568,6 +568,16 @@ MAU_DF2XSS_DFVSC2X_DSP_HTZC:
   - DSP
   - HTZC
 
+MAU_DFSS_DFVSC_HPDDC_ZTM:
+  description: "Dual discharge setpoint control with zone monitoring for heat pump MAU."
+  is_canonical: true
+  implements:
+  - MAU
+  - DFSS
+  - DFVSC
```

Comment:
How does it do speed control, and why is that not part of this dataset?

Reply:
It has linked speed control with a Melink KEF on the same control program. Not sure what you mean about it not part of this dataset.

Reply:
ah, so the melink sends it a speed setpoint to run at.